controllers/fenceagentsremediation_controller.go

```diff
@@ -27,8 +27,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
-	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
+	farV1alpha1 "github.com/medik8s/fence-agents-remediation/api/v1alpha1"
```

Comment:
Nit: what is the reason for this change ?

Reply:
Only my preferance. I though it is much more clear this way, but I can revert it